### PR TITLE
docs: update readme add icons to title

### PR DESCRIPTION
**Pull request by ibrataha8**
**Checklist:**
- [ ] Securely signed commits
- [ ] Component(s) with PHPDoc blocks, only if necessary or adds value
- [ ] Unit testing, with >80% coverage
- [ ] User guide updated
- [ ] Conforms to style guide

### CODE REVIEW

**Review by datamweb — COMMENTED** [review]
Personally, I would like to see whether this PR will be merged or not. Similar to this in Laravel.

In any case, it needs to be signed.
https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/pull_request.md#signing

**Review by MGatner — COMMENTED** [review]
A quick read doesn't bring up anything definitive about ASCII versus UTF-8 specs for Markdown. I'm not sure what the implications of this are but it seems a common practice and I'll about some emojis 😅😉🙃🫶

**Comment by ibrataha8** [comment]
@MGatner Thank You

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# CodeIgniter 4 Development
+# CodeIgniter 4 Development
```
Comment (kenjis) [inline README.md:1]: Please remove the unneeded space at the end.

File: README.md [hunk 2]
```diff
@@ -69,7 +69,7 @@ into the framework roadmap.
 Remember that some components that were part of CodeIgniter 3 are being moved
 to optional packages, with their own repository.
 
-## Contributing
+## Contributing 🤝
```
Comment (kenjis) [inline README.md:72]:
```suggestion
## Contributing 🤝
```

**Comment by kenjis** [comment]
We can of course use UTF-8 in Markdown. These emojis make sense. 
I'm fine. But we need your GPG sign.
https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/pull_request.md#signing

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ibrataha8) [reply README.md:1]: i fixed thank you

[the remark this answers is not in the document]
Comment (ibrataha8) [reply README.md:1]: i fixed thank you

**Comment by ibrataha8** [comment]
@kenjis If you like my addition, don't forget to add it. I'm just improving user experience

**Review by kenjis — APPROVED** [review]

**Comment by kenjis** [comment]
@ibrataha8 I had overlooked the fact that the commit was not signed.

You must [GPG-sign](https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/signing.md) your work.
See https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/pull_request.md#signing

And there are two more README. See https://github.com/codeigniter4/CodeIgniter4/pull/7732#issuecomment-1652217870
So it would be nice that you add emojis to them in this PR.

**Comment by kenjis** [comment]
See https://github.com/codeigniter4/shield/pull/983

**Pull request closed without merging**